src/main/java/com/minecolonies/coremod/entity/ai/citizen/baker/BakerRecipes.java

```diff
@@ -0,0 +1,98 @@
+package com.minecolonies.coremod.entity.ai.citizen.baker;
+
+import com.minecolonies.coremod.entity.ai.util.RecipeStorage;
+import net.minecraft.init.Items;
+import net.minecraft.item.ItemStack;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Contains all recipes the baker can use.
+ */
+public final class BakerRecipes
+{
+    /**
+     * The grid size the baker can use.
+     */
+    private static final int GRID_SIZE = 9;
+
+    /**
+     * Amount of wheat required for recipe.
+     */
+    private static final int REQUIRED_WHEAT = 3;
+
+    /**
+     * Amount of milk buckets required for the recipe.
+     */
+    private static final int REQUIRED_MILK = 3;
+
+    /**
+     * Amount of sugar required for a recipe.
+     */
+    private static final int REQUIRED_SUGAR = 2;
+
+    /**
+     * Amount of egg required for a recipe.
+     */
+    private static final int REQUIRED_EGGS = 1;
+
+    /**
+     * Amount of cocoa required for a recipe.
+     */
+    private static final int REQUIRED_COCOA = 1;
+
+    /**
+     * Amount of wheat required for a cookie recipe.
+     */
+    private static final int REQUIRED_WHEAT_COOKIES = 2;
+
+    /**
+     * List of recipes the Baker can know.
+     */
+    private static final List<RecipeStorage> recipes = new ArrayList<>();
+
+    /**
+     * Amount of buckets he should give back after a cake
+     */
+    private static final int BUCKET_COUNT = 3;
+
+    static
+    {
+        final List<ItemStack> inputCake = new ArrayList<>();
```

Comment:
In future iterations of the baker, maybe we should use forge to determine the recipe ingredients.